bin/clover/src/pipelines/gcp/funcs.ts

```diff
@@ -42,7 +42,24 @@ export const ACTION_FUNC_SPECS = {
   FuncSpecInfo & { actionKind: ActionFuncSpecKind }
 >;
 
-export const CODE_GENERATION_FUNC_SPECS = {} as const satisfies Record<
+export const CODE_GENERATION_FUNC_SPECS = {
+  "GCP Create Code Gen": {
+    id: "0000000000000000000000000000000000000008",
```

Comment:
Just a note, we MUST generate the appropriate ids for these before these get shipped to the module index